privacyidea/static/components/config/views/config.periodictasks.details.html

```diff
@@ -80,6 +80,20 @@
         </div>
     </div>
 
+    <div class="form-group" ng-show="ptaskid">
+        <label for="retry_if_failed" class="col-sm-3 control-label"
+        translate>Retry If Faild</label>
+        <div class="col-sm-6">
+            <input type="checkbox" id="retry_if_failed" name="retry-periodic-task"
+            ng-model = "form.retry_if_failed">
+            <p class="help help-block" translate>
+                If retry if failed is true, the task will be retry by failing,
```

Comment:
"retry by failing" sounds a bit awkward to me. (See above).

Please change this in all three places to s.th. like "privacyIDEA will retry to execute the task if it failed..."

Reply:
Done in 969450a5